trimmer_decoraator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative 'decorator'
+
+class TrimmerDecorator < Decorator
+  def correct_name
+    @nameable.name.strip.slice(0, 10) if @nameable.name.length > 10
```

Comment:
- [ ] Kindly implement this just as I have done for the Capitalize class. 

![image](https://user-images.githubusercontent.com/86318284/199226065-520f65f4-052a-445a-99f5-e21376bba506.png)